plugins/blip/basics/components/chart/BasalBolusRatio.js

```diff
@@ -62,26 +76,49 @@ var BasalBolusRatio = React.createClass({
   render: function() {
     var data = this.props.data;
     var percent = d3.format('%');
+    var basal = _.get(data, ['basalBolusRatio', 'basal'], null);
+    var bolus = _.get(data, ['basalBolusRatio', 'bolus'], null);
+    var basalPercentClasses = cx({
+      'BasalBolusRatio-percent': true,
+      'BasalBolusRatio-percent--basal': !!basal,
+      'BasalBolusRatio--nodata': !basal
+    });
+    var basalLabelClasses = cx({
+      'BasalBolusRatio-label': true,
+      'BasalBolusRatio-label--basal': !!basal,
```

Comment:
Yah, this is one of the accepted ways to coerce to a Boolean in JavaScript - `!!x` is the equivalent of `Boolean(x)` (but requires less typing 😜). I first learned it from Gordon, and @krystophv and I talked about it recently on one of his PRs, and I think we decided we like it generally. (@krystophv chime in if that's not the takeaway you remember from our exchange.)